### PR TITLE
Fixes Dev listings button color in pink theme

### PR DESCRIPTION
**Pull request by Aswathprabhu**
Fixes Dev listings button color in pink theme

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Related Tickets & Documents

Closes https://github.com/thepracticaldev/dev.to/issues/3667

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by Aswathprabhu** [comment]
Fixes the issue [3667](https://github.com/thepracticaldev/dev.to/issues/3667) 🍾

**Comment by rhymes** [comment]
@Aswathprabhu thanks for your contribution, can you provide a screenshot just in case? thanks!

**Comment by Aswathprabhu** [comment]
> @Aswathprabhu thanks for your contribution, can you provide a screenshot just in case? thanks!

Sure @rhymes 

Shot

File: app/assets/stylesheets/classified_listings.scss [hunk 1]
```diff
@@ -556,6 +556,11 @@
     theme-color,
     $white
   );
+  @include themeable(
+    background-color,
+    theme-container-accent-background,
+    $white
```
Comment (Glennmen) [inline app/assets/stylesheets/classified_listings.scss]: This doesn't seem right 🤔Both default value for color and background-color are `$white`

Comment (Aswathprabhu) [reply app/assets/stylesheets/classified_listings.scss]: @Glennmen thanks for spotting this! I've fixed this now 👍

**Pull request merged**